lib/facil/fio.c

```diff
@@ -307,7 +307,7 @@ static inline int fio_clear_fd(intptr_t fd, uint8_t is_open) {
   if (fio_data->max_protocol_fd < fd) {
```

Comment:
Perhaps this is the line we need to fix by stating `if (is_ope && fio_data->max_protocol_fd < fd)`...?

Reply:
Alright, i will try

Reply:
Tested and changed